includes/admin/settings/display-settings.php

```diff
@@ -21,11 +21,43 @@ function edd_admin_header() {
 	if ( ! edd_is_admin_page( '', '', false ) ) {
 		return;
 	}
+
+	$numberNotifications = EDD()->notifications->countActiveNotifications();
 	?>
 	<div id="edd-header" class="edd-header">
-		<img class="edd-header-logo" alt="" src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg' ); ?>" />
+		<div id="edd-header-wrapper">
+			<img class="edd-header-logo" alt="" src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg' ); ?>" />
+
+			<div id="edd-header-actions">
+				<button
```

Comment:
I was just about to post if maybe we should allow dismissed notifications to be visible. Perhaps I dismiss the purchase a pass today but want to see it again or use the buttons in a week.

Reply:
I do like the idea, but may save that for a future release just because it's not part of the requirements. :grin: But I do like the option of having it in the future.